lib/logstash/filters/fingerprint.rb

```diff
@@ -123,7 +123,12 @@ def anonymize_openssl(data)
   def anonymize_murmur3(value)
     case value
     when Fixnum
-      MurmurHash3::V32.int_hash(value)
+      if x64?
+        MurmurHash3::V32.int32_hash(value)
```

Comment:
This conditional is confusing. it asks if we are on 64bit, and if so, does the 32bit hash, otherwise does the 64bit hash.

Seems inverted?